CMakeLists.txt

```diff
@@ -451,33 +442,73 @@ if(MSVC)
   )
 endif()
 
+#
+# Find boost
+if(BUILD_WITH_BOOST)
+  hunter_add_package(Boost)
+  find_package(Boost CONFIG REQUIRED)
+  target_link_libraries(${LIB_NAME} Boost::boost)
+endif()
+
+#
+# Find Expat
+if(ARABICA_XML_BACKEND STREQUAL USE_EXPAT)
+  hunter_add_package(Expat)
+  find_package(EXPAT REQUIRED)
+  target_link_libraries(${LIB_NAME} ${EXPAT_LIBRARIES})

```

Comment:
There is an imported target:
- https://github.com/ruslo/hunter/blob/v0.23.155/examples/Expat/CMakeLists.txt#L18

We should use it. The documentation page seems to be out-of-date:
- https://docs.hunter.sh/en/latest/packages/pkg/Expat.html

Feel free to update :)